mkdocs.yml

```
@@ -1,20 +1,95 @@
 site_name: Guidelines and Documentation
 pages:
```

The 'pages' configuration option has been deprecated and will be removed in a future release of MkDocs. Use 'nav' instead. 

```suggestion
nav:
```